### DISCUSSION TITLE
Brownfield land

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#28] [Planning consideration]
# Brownfield land

Brownfield land is the set of brownfield sites a local planning authority has assessed as appropriate for residential development.

## What we've done

[Legislation from 2017](https://www.legislation.gov.uk/uksi/2017/403/regulation/3/made) stipulated that Local planning authorities need to publish a register of their brownfield sites each year. There is [guidance on GOV.UK](https://www.gov.uk/guidance/brownfield-land-registers) that goes into more detail about brownfield land.

This was one of the first datasets we looked at because we knew data existed, yet no national dataset was available. We decided we should be able to create one. However, when we tried, we soon realised the published data was inconsistent, and it wasn't trivial to clean and aggregate.

To improve the situation, we iterated the existing Standard and wrote and tested new guidance. We also built the beings of the planning data platform to do some of the heavy lifting; fixing common formatting issues, mapping from the old to the new version of the standard and aggregating the data into a national dataset.

You can read the [guidance for the Brownfield Land Standard on GOV.UK](https://www.gov.uk/government/publications/brownfield-land-registers-data-standard), it also includes the original version.

The [platform currently collects data from 346 different publishers](https://www.planning.data.gov.uk/dataset/brownfield-land).

## What we're looking for

We'd like to know:

* how the standard works for you, are there any confusing bits?
* what is missing from the standard?
* how is the data being used?
* where is it being used?
* what would improve brownfield land data?
* anything else you think we should know?

**Comment by josh-keys** [comment]
We’re using the data to present development opportunities to clients. It’s the kind of information that they have asked for many times and find beneficial.

The principal issue we’ve encountered is poor validation of coordinates. Many records appear in the sea or in Ireland due to swapped latitudes and longitudes, or even eastings and northings having been swapped before being converted to latitude and longitude. Some records (1,769 of those that have no end date) have no coordinates whatsoever. We managed to relocate 660 records that were incorrectly placed or had no coordinates at all, through a mixture of reversing the transformation of coordinates or through use of the address.

Some records have quite detailed addresses but many do not, sometimes because they’re essentially non-addressable plots of land. Without coordinates or a good address it’s very difficult to locate the sites, which means we cannot present the opportunity to users.

![Screenshot 2023-10-17 at 14 39 49](https://github.com/digital-land/data-standards-backlog/assets/108812796/f3e53190-d933-48d6-8474-82d37e7f1285)

**Reply by drewshardy** [reply]
Further to this, an additional constraint in using this data is that for maximum utility, polygon data is required. I'm aware the original standards mandated spatial polygon data to be available for all sites, but the standard was sadly dropped. 

Developers need to know how brownfield land interacts with land parcels and with additional constraints and historic mapping etc, so point data can only take them so far down the path of properly assessing an opportunity. All LPAs have the polygon data _somewhere_, so I would welcome an initiative to free it from closed systems if we have a serious commitment to brownfield first development.

**Reply by colmjude** [reply]
Hi both. Thanks for your comments. It's really helpful to see what's not working for people.

We've noticed that too @josh-keys, and we've been working with local planning authorities to understand what's happening and improve the data. Coordinates the wrong way around and conversion (OSGB36 to WGS84) are the main culprits. We are also exploring some improved validation tools. We last published [weeknotes](https://digital-land.github.io/data-standards/weeknote/) a while ago but we will soon, and they should explain more about what we are doing on this front. Hopefully, the data will start to improve soon.

@drewshardy, The current version of the brownfield land standard, published in 2019, doesn't include the polygon. Which, as you say, limits how valuable this dataset is. It would be great if we could update the standard to include the polygon data, but it's complicated by [the legislation listing out what should be included](https://www.legislation.gov.uk/uksi/2017/403/schedule/2/made). The legislation is hard/time-consuming to change. However, we have spec'd out [a brownfield-site schema](https://raw.githubusercontent.com/digital-land/specification/main/content/dataset/brownfield-site.md) that includes the polygon (the `geometry` field). A few LPAs have started testing it, there are [> 2500 records](https://www.planning.data.gov.uk/dataset/brownfield-site) with polygons on planning.data.gov.uk.
Can you expand on your point above and explain what sort of thing you'd be able to do if you had the polygon data? The more evidence we have about people needing the data the easier it is for us to prioritise more work on it.

**Reply by drewshardy** [reply]
Thanks @colmjude. To expand upon my point above, we work in the proptech sector, specifically in the developer space. We help developers identify development opportunities, constraints, planning data and local policy to enable them to find the best sites for development. This is done within a SaaS platform, which is primarily map-driven. 
1) With polygon data for brownfield land, a client will be able to identify which title polygons intersect the site, and therefore who to contact in order to start the process. 
2) we have an experimental tool which calculates remediation costs and issues, but this is area-based and therefore can only work with polygon data. This could provide key information on the viability of brownfield sites, increasing the potential for them being developed.
3) Polygon data allows the sites to be assessed against spatial policy and constraints e.g. flood risk etc, thereby increasing the certainty a client has regarding the development potential for a site
Hope this helps, but I'm sure we could come up with a few more if you need it!

**Comment by colmjude** [comment]
## Difficult to update the specification

The fields contained in the brownfield land specification are captured in the legislation around the brownfield land register. This makes it much harder to change fields, as each change requires a change to the legislation.
